### PR TITLE
Update README.md to make it easier to understand for novices.

### PR DESCRIPTION
**Pull request by nielsdrost**
This PR tries to make the README a bit more accessible to novices. I mostly added some introduction text here and there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +28,11 @@ In addition to the standard VM storage, additional read-only datasets are mounte
 * parameter-sets, example use cases and bigger ones
 * singularity-images of hydrological models wrapped in grpc4bmi servers
 
-## Local test VM
+Previously the eWatercycle platform consisted of multiple VM on SURF HPC cloud, see [v0.1.2 release](https://github.com/eWaterCycle/infra/releases/tag/v0.1.2) for that code.
 
-This chapter is dedicated for application developers.
+## Setup of eWaterCycle platform on a local test VM
+
+Deplying a local test VM is mostly useful for developing the SURF Research Cloid applications. This vagrant setup should work on any Linux or Windows machine with sufficient memory (8Gb, mostly depends on the models you run), cpu (More is better, one core will do if needed), and storage (At least 200Gb) available.
```
Comment (nielsdrost) [inline README.md:35]: Can someone verify these numbers?

Comment (RolfHut) [reply README.md:35]: verified. These numbers make sense.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+This repo contains (codified) instructions for deploying the eWaterCycle platform. The target audience of these instructions are system administrators. For more information on the eWaterCycle platform (and how to deloy it) see the [eWaterCycle documentation[(https://ewatercycle.readthedocs.io/).
+
+For instructions on how to use the machine as deployed by this repo see the [User guide](USER.md).
+
+These instructions assume you have some basic knowledge of [vagrant](https://vagrantup.com) and
+ [Ansible](https://docs.ansible.com/ansible/latest/index.html).
+
+## Setup of eWaterCycle platform on the SURF Research cloud
+
+De environment used by the eWaterCycle platform development team is the [SURF Research Cloud](https://servicedesk.surfsara.nl/wiki/display/WIKI/Research+Cloud+Documentation). The setup instructions in this repo will create an eWatercycle application(a sort-of VM template) that when started will create a machine with:
```
Comment (RolfHut) [inline README.md:15]: typo (de instead of the)

File: README.md [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
 
 * Explorer: web visualization of available models / parameter sets combinations and a way to generate Jupyter notebooks
 * Jupyter Hub: to interactivly generate forcings and perform experiments on hydrological models using the [eWatercycle Python package](https://ewatercycle.readthedocs.io/)
 * ERA5 and ERA-Interim global climate data, which can be used to generate forcings
 * Installed models and their example parameter sets
```
Comment (RolfHut) [inline README.md:21]: Maybe explain briefly that the one setting up the machine needs to have research budget, but can distribute access to the machine to others (using SRAM)?

File: README.md [hunk 4]
```diff
@@ -25,9 +28,11 @@ In addition to the standard VM storage, additional read-only datasets are mounte
 * parameter-sets, example use cases and bigger ones
```
Comment (RolfHut) [inline README.md:28]: bigger ones is unclear, remove?

**Review by RolfHut — COMMENTED** [review]
see the inline comments

File: README.md (hunk 3 above)
Comment (nielsdrost) [reply README.md:23]: Added a note.

File: README.md (hunk 2 above)
Comment (nielsdrost) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (nielsdrost) [reply README.md]: Removed

File: README.md [hunk 5]
```diff
@@ -1,33 +1,40 @@
...
+* parameter-sets
 * singularity-images of hydrological models wrapped in grpc4bmi servers
 
-## Local test VM
+Previously the eWatercycle platform consisted of multiple VM on SURF HPC cloud, see [v0.1.2 release](https://github.com/eWaterCycle/infra/releases/tag/v0.1.2) for that code.
 
-This chapter is dedicated for application developers.
+## Setup of eWaterCycle platform on a local test VM
+
+Deplying a local test VM is mostly useful for developing the SURF Research Cloid applications. This vagrant setup should work on any Linux or Windows machine with sufficient memory (8Gb, mostly depends on the models you run), cpu (More is better, one core will do if needed), and storage (At least 200Gb) available.
```
Comment (Peter9192) [inline README.md:37]:
```suggestion
Deploying a local test VM is mostly useful for developing the SURF Research Cloud applications. This vagrant setup creates a virtual machine with 8Gb memory, 4 virtual cores, and 70Gb storage. This should work on any Linux or Windows machine.
```

File: README.md [hunk 6]
```diff
@@ -1,33 +1,40 @@
-# Setup of eWaterCycle system on SURF Research cloud
+# Instructions for system administrators to deploy the eWaterCycle platform
 
 ![Ansible Lint](https://github.com/eWaterCycle/infra/workflows/Ansible%20Lint/badge.svg)
 [![Concept DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1462548.svg)](https://doi.org/10.5281/zenodo.1462548)
 
-On the [SURF Research Cloud](https://servicedesk.surfsara.nl/wiki/display/WIKI/Research+Cloud+Documentation) an eWatercycle application will be available that when started will give
+This repo contains (codified) instructions for deploying the eWaterCycle platform. The target audience of these instructions are system administrators. For more information on the eWaterCycle platform (and how to deloy it) see the [eWaterCycle documentation[(https://ewatercycle.readthedocs.io/).
```
Comment (Peter9192) [inline README.md:6]:
```suggestion
This repo contains (codified) instructions for deploying the eWaterCycle platform. The target audience of these instructions are system administrators. For more information on the eWaterCycle platform (and how to deploy it) see the [eWaterCycle documentation[(https://ewatercycle.readthedocs.io/).
```

File: README.md [hunk 7]
```diff
@@ -1,33 +1,40 @@
...
+For instructions on how to use the machine as deployed by this repo see the [User guide](USER.md).
+
+These instructions assume you have some basic knowledge of [vagrant](https://vagrantup.com) and
+ [Ansible](https://docs.ansible.com/ansible/latest/index.html).
+
+## Setup of eWaterCycle platform on the SURF Research cloud
+
+The hardware environment used by the eWaterCycle platform development team is the [SURF Research Cloud](https://servicedesk.surfsara.nl/wiki/display/WIKI/Research+Cloud+Documentation). Starting a machine on the Surf Research Cloud requires that you have research budget with SURF, for more info see the website of [SURF](https://www.surf.nl/en/research-it/apply-for-access-to-compute-services). Once running, access to the machine can be shared to anyone.
+
+The setup instructions in this repo will create an eWatercycle application(a sort-of VM template) that when started will create a machine with:
```
Comment (Peter9192) [inline README.md:17]:
```suggestion
The setup instructions in this repo will create an eWaterCycle application (a sort-of VM template) that when started will create a machine with:
```

**Review by Peter9192 — APPROVED** [review]
Nice work.

**Comment by Peter9192** [comment]
This PR was merged without committing the latest changes. I'll try to reopen it.

**Pull request merged**